### PR TITLE
Let the user view an event from the list on the map.

### LINKED ISSUE #380: Connect the list view to the bottom sheet on the map

### PR DESCRIPTION
**Pull request by alejandrocalles**
Should be able to click on  the map icon in an event on the Home Screen and view it on the map.

Closes #380.

### CODE REVIEW

**Comment by alejandrocalles** [comment]
Clicking on an event's 'view on map' button also centers the map on the event.

File: app/src/main/java/com/github/swent/echo/compose/map/MapViewProviderImpl.kt [hunk 1]
```diff
@@ -78,6 +78,11 @@ class MapLibreMapViewProvider : IMapViewProvider<MapView> {
         }
     }
 
+    override fun setSavedCameraPosition(newPosition: LatLng, zoomLevel: Double) {
+        lastCameraPosition =
+            CameraPosition.Builder().target(newPosition).zoom(zoomLevel).bearing(0.0).build()
```
Comment (srsingh04) [inline app/src/main/java/com/github/swent/echo/compose/map/MapViewProviderImpl.kt:83]: Knitpick: Isn't .bearing(0.0) redundant here? Is the default value of the bearing not 0?

**Review by srsingh04 — APPROVED** [review]
Thank you for working on this enhancement on the maps. It would really improve user experience of our application. 
Ready to be merged!

File: app/src/main/java/com/github/swent/echo/compose/map/MapViewProviderImpl.kt (hunk 1 above)
Comment (alejandrocalles) [reply app/src/main/java/com/github/swent/echo/compose/map/MapViewProviderImpl.kt:83]: You're right, I'll change that :)

**Review by srsingh04 — APPROVED** [review]
LGTM! Ready to be merged

**Pull request merged**